interpreter/python/python.go

```diff
@@ -655,33 +656,39 @@ func (d *pythonData) readIntrospectionData(ef *pfelf.File, symbol libpf.SymbolNa
 
 // decodeStub will resolve a given symbol, extract the code for it, and analyze
 // the code to resolve specified argument parameter to the first jump/call.
-func decodeStub(ef *pfelf.File, addrBase libpf.SymbolValue, symbolName libpf.SymbolName,
-	argNumber uint8) libpf.SymbolValue {
-	symbolValue, err := ef.LookupSymbolAddress(symbolName)
+func decodeStub(
+	ef *pfelf.File,
+	memoryBase libpf.SymbolValue,
+	symbolName libpf.SymbolName,
+) (libpf.SymbolValue, error) {
+	codeAddress, err := ef.LookupSymbolAddress(symbolName)
 	if err != nil {
-		return libpf.SymbolValueInvalid
+		return libpf.SymbolValueInvalid, fmt.Errorf("lookup %s failed: %w",
+			symbolName, err)
 	}
 
 	code := make([]byte, 64)
-	if _, err := ef.ReadVirtualMemory(code, int64(symbolValue)); err != nil {
-		return libpf.SymbolValueInvalid
+	if _, err = ef.ReadVirtualMemory(code, int64(codeAddress)); err != nil {
+		return libpf.SymbolValueInvalid, fmt.Errorf("reading %s 0x%x code failed: %w",
+			symbolName, codeAddress, err)
 	}
-
-	value := decodeStubArgumentWrapper(code, argNumber, symbolValue, addrBase)
+	value, err := decodeStubArgumentWrapper(code, codeAddress, memoryBase)
 
 	// Sanity check the value range and alignment
-	if value%4 != 0 {
-		return libpf.SymbolValueInvalid
+	if err != nil || value%4 != 0 {
+		return libpf.SymbolValueInvalid, fmt.Errorf("decode stub %s 0x%x %s failed (0x%x, %w)",
```

Comment:
The reason for the error should not be within the brackets.
```suggestion
		return libpf.SymbolValueInvalid, fmt.Errorf("decode stub %s 0x%x %s failed (0x%x):  %v",
```